.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -7,12 +7,16 @@ assignees: ''
 
 ---
 
+**NOTE: Items marked `jdkxx` and `TEMPLATE_UPDATEME` should be replaced while deploying this issue template. It is recommented to delete this line once you've done so :-)**
```

Comment:
```suggestion
**NOTE: Items marked `jdkxx` and `TEMPLATE_UPDATEME` should be replaced while deploying this issue template. It is recommended to delete this line once you've done so :-)**
```

Reply:
Changed - thanks (For some reason it didn't let me directly commit your suggestion though)